AdminApi/src/AdminApi/ClientApp/src/app/components/quotas/identity/identity-details/identity-details.component.ts

```diff
@@ -93,12 +93,29 @@ export class IdentityDetailsComponent {
         this.tierService.getTiers().subscribe({
             next: (tiers) => {
                 this.tiers = tiers.result;
+                if (this.identity.tierId !== "TIR00000000000000001") {
+                    this.tiers = this.tiers.filter((tier) => tier.id !== this.getQfDTierId());
+                }
                 this.updatedTier = this.tiers.find((t) => t.id === this.identity.tierId);
                 this.tier = this.updatedTier;
             }
         });
     }
 
+    public enablesManualAssignment(tier: TierOverview): boolean {
+        return tier.canBeManuallyAssigned;
+    }
+
+    public disabledTiers(tier: TierOverview): boolean {
```

Comment:
This method name isn't a method name. Method names should contain verbs... "disabledTiers" sounds as if it was a list of disabled Tiers

Reply:
Changed.